content/sql/concepts/operators/terms/update/update.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: UPDATE
+Description: 'Changes/Updates data row entries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Operators'
+  - 'Database'
+  - 'Queries'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The **`UPDATE`** operator changes data entries  .
+
+## Syntax
+
+```pseudo
+UPDATE table_name
+SET column_name = ... ;
+WHERE item_name = ...
+```
```

Comment:
```suggestion
```pseudo
UPDATE table_name
SET column_name = ... 
WHERE item_name = ... ;
```
```
In SQL, the `WHERE` clause should be placed before the semicolon.